File: src/net/eduard/api/lib/menu/PagedMenu.java
package net.eduard.api.lib.menu;

import org.bukkit.entity.Player;
import org.bukkit.inventory.Inventory;

public interface PagedMenu {

	public Inventory open(Player player, int page);

	public default Inventory open(Player player) {
		return open(player, 1);
	}
	public int getPageOpen(Player player);
	public boolean isOpen(Player player);
	

}

File: src/net/eduard/api/lib/menu/PagedMenu.kt
package net.eduard.api.lib.menu

import org.bukkit.entity.Player
import org.bukkit.inventory.Inventory

interface PagedMenu {

    fun open(player: Player, page: Int): Inventory

    fun open(player: Player): Inventory {
        return open(player, 1)
    }

    fun getPageOpen(player: Player): Int
    fun isOpen(player: Player): Boolean


}
